src/System.Data.SqlClient/src/System/Data/SqlClient/SqlParameterCollection.cs

```diff
@@ -30,6 +30,21 @@ internal bool IsDirty
                 _isDirty = value;
             }
         }
+        override public bool IsFixedSize
+        {
+            get
+            {
+                return ((System.Collections.IList)InnerList).IsFixedSize;
+            }
+        }
+
+        override public bool IsReadOnly
```

Comment:
Nit: This can be condensed to  `public override bool IsReadOnly => ((System.Collections.IList)InnerList).IsReadOnly;`

Reply:
Updated in the new commit.